third_party/2and3/mypy_extensions.pyi

```diff
@@ -1,8 +1,26 @@
-from typing import Dict, Type, TypeVar, Optional, Union, Any, Generic
+import abc
+import sys
+from typing import (
+    Dict, Type, TypeVar, Optional, Union, Any, Generic, Mapping, ItemsView, KeysView, ValuesView
+)
 
 _T = TypeVar('_T')
 _U = TypeVar('_U')
 
+# Internal mypy fallback type for all typed dicts (does not exist at runtime)
+class _TypedDict(Mapping[str, object], metaclass=abc.ABCMeta):
```

Comment:
Why this needs to be abstract?

Reply:
Stubs must declare all ABCs. The class is abstract since it doesn't include implementations of `Mapping` methods.